tests/webapp/api/test_objectstore_api.py

```diff
@@ -120,7 +120,7 @@ def test_objectstore_with_bad_key(job_sample, jm):
     tjc.add(tj)
 
     resp = test_utils.post_collection(
-        jm.project, tjc, status=403, consumer_key='wrong key'
+        jm.project, tjc, status=403, consumer_key='wrong-key'
```

Comment:
Since I'm using the consumer-key as part of a cache key, it cannot contain blank spaces, otherwise memcached will complain